src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -127,6 +129,8 @@ public PhpGapicSurfaceTransformer(ApiConfig apiConfig, GapicCodePathMapper pathM
         pathTemplateTransformer.generatePathTemplateGetterFunctions(context));
     xapiClass.pageStreamingDescriptors(pageStreamingTransformer.generateDescriptors(context));
     xapiClass.hasPageStreamingMethods(context.getInterfaceConfig().hasPageStreamingMethods());
+    xapiClass.bundlingDescriptors(ImmutableList.<BundlingDescriptorView>of());
+    xapiClass.hasBundlingMethods(context.getInterfaceConfig().hasBundlingMethods());
```

Comment:
Don't add these lines if the PHP snippet isn't using these fields... If you remove them, then also `hasBundlingMethods` no longer needs to be added to `DynamicLangXApiView`.